web_embedding/README.md

```diff
@@ -9,3 +9,25 @@ This directory contains examples of how to embed Flutter in web apps (without if
 
 Check the `README.md` of each example for more details on how to run it, and the
 "Points of Interest" it may contain.
+
+## Community Contributions
+
+Members of the community have graciously ported and contributed the following examples
+of Flutter Web embedding into other web frameworks:
+
+| Author | Host Framework | Code |
+|--------|----------------|------|
+| [@p-mazhnik][] | **React JS** | [p-mazhnik/flutter-embedding cra-flutter][] |
+| [@p-mazhnik][] | **React Native** | [p-mazhnik/flutter-embedding expo-flutter][] |
```

Comment:
I wonder if this sample could be mentioned in the `add_to_app` directory as well. It is an example of embedding Flutter into another cross-platform framework, React-Native (Android, iOS, web). So it demonstrates the possibility of gradual migration from this framework to Flutter, which can be valuable for React-Native developers. 

What are your thoughts on this, @domesticmouse ?

Reply:
@RedBrogdon we need to think about how to merge our Add to App story with this Flutter for web capability.